test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -688,6 +688,49 @@
 #      initPods: 20000
 #      measurePods: 5000
 
+# Measure throughput of regular schedulable pods that are interleaved by high priority preemption pods scheduled at 20/s rate.
+# Implementation of asynchrounous preemption feature https://github.com/kubernetes/kubernetes/issues/126858 is supposed to increase the trhoughput of the measured pods as all heavy operations (apiserver communication) will be performed asyncronously, wihtout blocking the scheduler loop.
+# How is it achieved:
+# 1. There are X initial nodes with 4 low priority pods each, consuming 3.6 CPU out of the total 4 available.
+# 2. High priority preemption which need to preempt 3 of 4 low priority pods to fit (require 3 CPU).
+# 3. Measured pods are always schedulable, as they require 0.1 CPU only.
+- name: PreemptionAsync
+  workloadTemplate:
+  - opcode: createNodes
+    countParam: $initNodes
+  - opcode: createPods
+    countParam: $initPods
+    podTemplatePath: config/templates/pod-low-priority.yaml
+  - opcode: churn
+    mode: create
+    templatePaths:
+    - config/templates/pod-high-priority-large-cpu.yaml
+    intervalMilliseconds: 50
+  - opcode: createPods
+    countParam: $measurePods
+    podTemplatePath: config/templates/pod-default.yaml
+    collectMetrics: true
+  workloads:
+  - name: 5Nodes
+    labels: [integration-test, performance, short]
+    params:
+      initNodes: 5
+      initPods: 20
+      measurePods: 5
+  - name: 500Nodes
+    labels: [performance, short]
+    params:
+      initNodes: 500
+      initPods: 2000
+      measurePods: 500
+  - name: 5000Nodes
+    labels: [performance]
+    threshold: 200
```

Comment:
come to think that we should probably rename this field to `thresholdThroughput`, but it can be done in a separate PR.

Reply:
or throughputThreshold

Reply:
But, this threshold can be applied to other metrics too, right? 
Basically we should aim at the scheduling throughput though, sometimes, like the ones to measure the event handling throughputs which @/macsko is working on, we target other metrics.

Reply:
oh I see. The tricky part is that, in the case of throughput it would be "greater than or equal to" threshold, whereas for latency, it would be "less than or equal to".

So we might be needing more parameters. But then we can defer any changes for when we actually need them. The good thing is that this is not a user-facing API, so there is no risk of breaking backwards compatibility.

Reply:
Metric selector has the `expectLower` field already:
https://github.com/kubernetes/kubernetes/blob/e39571591d2917ca642f47ebca11253a91793c4b/test/integration/scheduler_perf/scheduler_perf.go#L317-L320

Reply:
I have found it actually difficult to measure anything other than throughput as start/stopCollectingMetrics does not measure the time precisely (but after encountering the first scheduled Pod).

IIUC there is no test that use ThresholdMetricSelector and/or measure latency already. Is there a case when we'd like to have it? If so, how to write it?